server/homebrew.api.js

```diff
@@ -168,6 +169,20 @@ const api = {
 		return res.status(200).send(brew.style);
 	},
 
+	getMeta : async (req, res, next)=>{
```

Comment:
I see what `getMeta` is doing, but I am concerned this basically means we have to request the brew twice when opening a page (once for the metadata and again inside the `sharePage.jsx`).

Not sure if there is a way to apply the metadata at the same time of the initial brew fetch from the `sharePage`, which would at least save a Mongo call.

But at a minimum, I think we can use `await api.getBrew('share', true)` so we only fetch the stub.

Reply:
I've kept this separate at this stage because I'm not sure whether the metadata information needs to be returned in the initial response in order for the meta tags to be picked up correctly for social media previews.

I think that you are likely correct about `getBrew('share', true)`.